patch-package
--- a/node_modules/react-apollo/test-utils.js
+++ b/node_modules/react-apollo/test-utils.js
@@ -939,6 +939,9 @@
             return React.createElement(ApolloProvider, { client: this.state.client }, this.props.children);
         };
         MockedProvider.prototype.componentWillUnmount = function () {
+            if (!this.state.client.queryManager) {
+                return;
+            }
             var scheduler = this.state.client.queryManager.scheduler;
             Object.keys(scheduler.registeredQueries).forEach(function (queryId) {
                 scheduler.stopPollingQuery(queryId);
